File: backend/currencyApi/Domain/Entities/Currency.cs
public class Currency
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Symbol { get; set; } // NOVO CAMPO
    public string Description { get; set; }
    public bool Status { get; set; }
    public Backing Backing { get; set; }
    public ICollection<History> Histories { get; set; }
}

File: frontend/src/features/currency/types/Currency.ts
export type Backing = "BRL" | "USD" | "JPY" | "EUR" | "CNY" | "ARS";

export interface History {
  id: number;
  datetime: string;
  price: number;
  currencyId: number;
}

export interface Currency {
  id?: number;
  name: string;
  symbol: string; // NOVO CAMPO
  description: string;
  status: boolean;
  backing: Backing;
  icon: string;
  histories?: History[];
}